docs/resources/app.md

```diff
@@ -26,11 +26,12 @@ EOF
 }
 
 resource "coder_app" "code-server" {
-  agent_id  = coder_agent.dev.id
-  name      = "VS Code"
-  icon      = data.coder_workspace.me.access_url + "/icons/vscode.svg"
-  url       = "http://localhost:13337"
-  subdomain = false
+  agent_id      = coder_agent.dev.id
+  name          = "VS Code"
+  icon          = data.coder_workspace.me.access_url + "/icons/vscode.svg"
+  url           = "http://localhost:13337"
+  sharing_level = "owner"
```

Comment:
What do you think about `access` as a name instead?

Reply:
so just `access = "owner"`?

Reply:
after talking in slack we decided on `share` instead